runelite-client/src/main/java/net/runelite/client/plugins/menuentryswapper/MenuEntrySwapperPlugin.java

```diff
@@ -468,6 +470,27 @@ private void swapMenuEntry(int index, MenuEntry menuEntry)
 			{
 				swap("start-minigame", option, target, index);
 			}
+
+			if (target.equals("trader crewmember"))
```

Comment:
Move this above `swapTrade` and `swapTravel` like the slayer assignment is so you don't need to do the `!target.equals("trader crewmemeber")` for those.

Reply:
Doing this change we get a side effect if the option `OFF` is selected where it will swap the trade option depending on what `swapTrade` is set to. Is that a behaviour we want? Doesn't occur with `swapTravel` as `charter` gets removed from those options.